tests/draft2019-09/default.json

```diff
@@ -45,5 +45,35 @@
                 "valid": true
             }
         ]
+    },
+    {
+        "description": "the default keyword does not do anything if the property is missing",
+        "schema": {
+            "type": "object",
+            "properties": {
+                "alpha": {
+                    "type": "number",
+                    "maximum": 3,
+                    "default": 5
+                }
+            },
+        ,
```

Comment:
Missed one.
```suggestion
            }
        },
```

Reply:
wat? no... oh, the last amendment I made never made it to the server.. :'(

Reply:
and why aren't we running these files against test-schema.json? :(